tanner/utils/patterns.py

```diff
@@ -4,7 +4,6 @@
 RFI_ATTACK = re.compile(r'.*((http(s){0,1}|ftp(s){0,1}):).*', re.IGNORECASE)
 LFI_ATTACK = re.compile(r'.*(/\.\.)*(home|proc|usr|etc)/.*')
 LFI_FILEPATH = re.compile(r'((\.\.|/).*)')
-XSS_ATTACK = re.compile(r'.*<(.|\n)*?>')
```

Comment:
Why did you remove it from here?

Reply:
Template injection payloads can be matched with xss regex but vice versa is not possible.